tests/integration/sdk_api/artifacts/test_artifact_tags.py

```diff
@@ -30,37 +30,41 @@ def test_list_artifact_tags(self):
         key = "some-key"
         data_frame = pandas.DataFrame({"x": [1, 2]})
         artifact = mlrun.artifacts.dataset.DatasetArtifact(key, data_frame)
-        uid = "some-uid"
```

Comment:
here again I think its worth to keep the old test as well to ensure BC

Reply:
But this doesn't test BC, it tests the artifact tags, and it will fail when we remove `uid` support completely.. is it critical?